README.md

```diff
@@ -3,25 +3,22 @@ A formatter for behave's outputs using Cucumber
 
```

Comment:
I (Informational): I think we should give credit/a shoutout to the original version [like we did in the `poetry-monorepo-dependency-plugin`](https://github.com/TechnologyBrewery/poetry-monorepo-dependency-plugin):
> Forked and inspired by the [poetry-stickywheel-plugin](https://github.com/artisanofcode/poetry-stickywheel-plugin), this [Poetry](https://python-poetry.org/) plugin facilitates the usage of more complex monorepo project structures by pinning version dependencies when building and publishing archives with local path dependencies to other Poetry projects within the same monorepo.